site/source/docs/building_from_source/toolchain_what_is_needed.rst

```diff
@@ -23,7 +23,7 @@ Emscripten tools and dependencies
 A complete Emscripten environment requires the following tools. First test to see if they are already installed using the :ref:`instructions below <toolchain-test-which-dependencies-are-installed>`. Then install any missing tools using the instructions in the appropriate platform-specific build topic (:ref:`building-emscripten-on-linux`, :ref:`building-emscripten-on-windows-from-source`, :ref:`building-emscripten-on-mac-osx-from-source`):
 
 	- :term:`Node.js` (0.8 or above; 0.10.17 or above to run websocket-using servers in node):
-	- :term:`Python` 2.x (2.7.3 or above preferred)
+	- :term:`Python` 2.7 (2.7.3 or above preferred)
```

Comment:
Do we want to update our preference here? Maybe 2.7.12?

Reply:
Good idea, done.